### PR TITLE
Highlight unpublished content

### PR DESCRIPTION
**Pull request by mconnell**
When selecting content for a feed, this change highlights unpublished content in red.

screenshot 2015-07-09 15 33 06

### CODE REVIEW

File: app/assets/stylesheets/components/_source_item.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.source-item--unpublished {
+  color: red;
```
Comment (spencer) [inline app/assets/stylesheets/components/_source_item.scss]: color: $colour-red-medium;

**Comment by spencer** [comment]
:+1:

**Comment by mconnell** [comment]
@spencer I've just added an additional commit that highlights unpublished content on the feeds#show page. So this will need to be re-reviewed. Sorry.

**Comment by mconnell** [comment]
(It was something spotted in the demo, so it made sense to fix it here)

**Comment by spencer** [comment]
Re-reviewed :+1:

**Comment by mconnell** [comment]
PR #76 was merged into master, so this branch has just been rebased to be up to date with master.

**Comment by darokel** [comment]
:+1: Left one comment but realised it was resolved in https://github.com/moneyadviceservice/webfeeds/commit/99b3a721670b18acfeb23098af67c2a9909a48c1

**Pull request merged**